config/app.php

```diff
@@ -135,6 +135,7 @@
 		'Illuminate\Translation\TranslationServiceProvider',
 		'Illuminate\Validation\ValidationServiceProvider',
 		'Illuminate\View\ViewServiceProvider',
+		'Illuminate\Broadcasting\BroadcastServiceProvider',
```

Comment:
I used spaces, but then fixed it to tabs to use the same indent style as the rest of the service providers.